lib/howitzer/utils/transformers.rb

```diff
@@ -0,0 +1,39 @@
+module Howitzer
+  module Utils
+    # This module adds transformers methods for cucumber step arguments
+    module Transformers
+      # Transforms factories or factory properties from table to real values
+      # @example
+      #  When I sign up with following data:
+      #   | name                 | email                 |
+      #   | FACTORY_USER1[:name] | FACTORY_USER1[:email] |
+      #  #=> build(:user).name and build(:user).email
+      # In step:
+      # Given 'example step' do |table|
+      #   table = transform_table(table)
+      #     ...
+      # end
+
+      def transform_table(table)
+        raw = process_table(table)
+        location = Cucumber::Core::Ast::Location.of_caller
```

Comment:
Potentially it will not work. Despite Cucumber is howitzer dependency it is specified as a development dependency. After publishing to rubygems.org it can generate error on `gem install howitzer`. It will be nice to keep cucumber dependency as it is, because otherwise it will be required to update Howitzer on each version of Cucumber

Reply:
it looks like cucumber does not support parameter types for tables (we also can not support it)
so, lets remove this file,
create bug in howitzer about tables transformations with links to related bugs in cucumber,
and described solution for tables transformation.